packages/app/src/components/uiBlocks/Textfield.vue

```diff
@@ -35,11 +32,8 @@ export default defineComponent({
       type: String,
       default: '',
     },
-    label: {
-      type: String,
-      default: '',
-    },
   },
+
   emits: ['update:modelValue'],
 });
-</script>
+</script>
```

Comment:
```suggestion
</script>

```
Just add a new line to fix the failed lint check